### PR TITLE
DAT-2880 add bottom border

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-2880

### CODE REVIEW

**Comment by drsnyder** [comment]
Per the [guidelines](https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md#guidelines) please add a description to your pull request.

File: extensions/wikia/PortableInfobox/styles/PortableInfobox.scss [hunk 1]
```diff
@@ -98,6 +98,11 @@ $infobox-section-header-background: mix($color-page, $color-links, 75%);
 		}
 	}
 
+	.item-type-group,
+	.item-type-comparison {
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/styles/PortableInfobox.scss]: thise selector is excatly the same .item-type-key-val:not(:last-of-type), maybe we should merge them

File: extensions/wikia/PortableInfobox/styles/PortableInfobox.scss [hunk 2]
```diff
@@ -76,10 +76,16 @@ $infobox-section-header-background: mix($color-page, $color-links, 75%);
 		line-height: 28px;
 	}
 
-	.item-type-key-val:not(:last-of-type) {
+	.item-type-key-val:not(:last-of-type),
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/styles/PortableInfobox.scss]: Do we still need this :not here?

**Comment by dianafa** [comment]
LGTM

File: extensions/wikia/PortableInfobox/styles/PortableInfobox.scss (hunk 2 above)
Comment (RafLeszczynski) [inline extensions/wikia/PortableInfobox/styles/PortableInfobox.scss]: yes, for example for last `<data>` inside the `<group>`

**Pull request merged**